jetbrains-ultimate/src-201+/software/aws/toolkits/jetbrains/services/rds/RdsResources.kt

```diff
@@ -35,16 +31,10 @@ object RdsResources {
         """.*//(.+)\..*\..*.rds\..""".toRegex()
 
     fun extractRegionFromUrl(url: String?): String? = url?.let { RDS_REGION_REGEX.find(url)?.groupValues?.get(1) }
-    fun extractIdentifierFromUrl(url: String?): String? = url?.let { RDS_IDENTIFIER_REGEX.find(url)?.groupValues?.get(1) }
-
-    val LIST_INSTANCES_MYSQL: Resource.Cached<List<DBInstance>> = listInstancesFilter(mysqlEngineType)
-    val LIST_INSTANCES_POSTGRES: Resource.Cached<List<DBInstance>> = listInstancesFilter(postgresEngineType)
-    val LIST_INSTANCES_AURORA_MYSQL: Resource.Cached<List<DBInstance>> = listInstancesFilter(auroraMysqlEngineType)
-    val LIST_INSTANCES_AURORA_POSTGRES: Resource.Cached<List<DBInstance>> = listInstancesFilter(auroraPostgresEngineType)
 
-    private fun listInstancesFilter(engine: String) = ClientBackedCachedResource(RdsClient::class, "rds.list_instances.$engine") {
-        this.describeDBInstancesPaginator {
-            it.filters(Filter.builder().name(engineFilter).values(engine).build())
-        }.toList().flatMap { it.dbInstances() }
+    val LIST_SUPPORTED_INSTANCES: Resource.Cached<List<DBInstance>> = ClientBackedCachedResource(RdsClient::class, "rds.list_supported_instances") {
```

Comment:
Guess this fixes the issue with 5.7 reported in #2124 (in respect to the explorer)

Reply:
Nvm, no it doesn't. Aurora Mysql has 2 engine values now, so we need to cope with that.